UKHomeOffice/sgmr-service
# Update 'Constant' files and file structure

### AC / Description of changes

Tidying up our constants and removing the lib file as this is outdated practice

### To Test

- run app
- you should be able to log in
- the app should function as expected
- unit tests should pass

### Notes

---

**Remember to merge to feature branches not `master`.**
